scripts/configure-installer-tests-cluster.sh

```diff
@@ -39,8 +39,100 @@ if [ -z $CI ]; then
 fi
 
 # Setup the cluster for Operator tests
-## First, enable a cluster-wide operator
-oc create -f - <<EOF
+
+## Create a new namesapce which will be used for OperatorHub checks
+oc new-project $CI_OPERATOR_HUB_PROJECT
+## Let developer user have access to the project
+oc adm policy add-role-to-user edit developer
+
+CI_SERVER_VERSION=$(oc version | awk '/Server/ {print $3}')
+
+## If we're running on 4.1, perform relevant steps
+
+if [ $CI_SERVER_VERSION == 4.1.0 ]; then
+    ### First, install cluster-wide operator
+    ### CatalogSourceConfig for mongodb
+    oc create -f -<<EOF
+apiVersion: operators.coreos.com/v1
```

Comment:
It would be better to save this into a separate file, this is unmaintainable :-/

Reply:
Agree. I'll do that. :+1: